### PR TITLE
Srinter 745

### PR DESCRIPTION
**Pull request by ooneill547**
- Mirror format options from D7.
- H1 no longer available.
- Add information notice style.
- Restrict HTML so pasting from Word is cleaner - this may need to be looked at further.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -127,6 +186,9 @@
         "patches": {
             "drupal/core": {
                 "Add contextual link to blocks in layout builder": "https://www.drupal.org/files/issues/2019-09-06/contextual_links_with_LB-3020876-18_0.patch"
+            },
+            "drupal/console": {
+                "Fix spelling error in plugin.php.twig": "https://github.com/ooneill547/drupal-console/commit/ef82d2548c34d21f5288ee59b322bc10dbdf753c.patch"
```
Comment (johangant) [inline composer.json:191]: If you get a few spare minutes, I'd consider opening a pull request at https://github.com/hechoendrupal/drupal-console to get some credit

Comment (ooneill547) [reply composer.json:191]: Already done. I don't think it's been accepted yet.

**Review by johangant — APPROVED** [review]
LGTM

**Review by DuttonMa — APPROVED** [review]
Nice work

**Pull request merged**